exchanges/gateio/gateio_test.go

```diff
@@ -3366,6 +3414,7 @@ func TestOrderbookChannelIntervals(t *testing.T) {
 	assert.Equal(t, "20ms", i)
 
 	for s, exp := range map[*subscription.Subscription]error{
+		{Asset: asset.Binary, Channel: "unhandle_random_ticker", Interval: kline.OneYear}:                            nil,
```

Comment:
🦍 Slightly uneasy about this. Iterated a few ideas before coming up with this.
 ```suggestion
		{Asset: asset.Binary, Channel: "unknown_channel", Interval: kline.OneYear}:                            nil,
```